api/services/dmStoreConstants.ts

```diff
@@ -0,0 +1 @@
+export const ERROR_UNABLE_TO_UPLOAD_DOCUMENT = 'Unable to upload document to Document Management (DM) store'
```

Comment:
lets have one constant as else we could have endless files for various constants

Reply:
Centralised errorConstants file now in lib